### PR TITLE
[DOCFIX] Update docs for mapreduce

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

File: docs/en/compute/Hadoop-MapReduce.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Start Alluxio locally:
 $ ./bin/alluxio-start.sh local SudoMount
 ```
 
-You can add a sample file to Alluxio to run wordcount on. From your Alluxio directory:
+You can add a sample file to Alluxio for run MapReduce wordcount on. From your Alluxio directory:
```
Comment (LuQQiu) [inline docs/en/compute/Hadoop-MapReduce.md:90]: to run?

**Review by LuQQiu — COMMENTED** [review]
Overall LGTM, left a small comment, thanks!

File: docs/en/compute/Hadoop-MapReduce.md (hunk 1 above)
Comment (gpang) [reply docs/en/compute/Hadoop-MapReduce.md]: updated

**Review by gpang — COMMENTED** [review]
@LuQQiu updated the PR

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks!

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**